framework/include/multiapps/MultiApp.h

```diff
@@ -461,8 +467,12 @@ class MultiApp : public MooseObject,
   /// The type of application to build
   std::string _app_type;
 
-  /// The positions of all of the apps
+  /// The positions of all of the apps (to be deprecated)
   std::vector<Point> _positions;
+  /// The positions of all of the apps, using the Positions system
+  std::vector<const Positions *> _positions_objs;
+  /// The offsets, in case multiple Positions objects are specified
+  std::vector<unsigned int> _positions_index_offsets;
 
   /// Toggle use of "positions"
```

Comment:
This doxygen seems ambiguous given the sweeping additions you're making. I might flesh this doxygen comment out a bit